x-pack/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/add_policy_to_template_confirm_modal.js

```diff
@@ -37,7 +37,7 @@ export class AddPolicyToTemplateConfirmModalUi extends Component {
     const policyName = policy.name;
     if (!templateName) {
       this.setState({ templateError: i18n.translate(
-        'xpack.indexLifecycleMgmt.policyTable.addLifecyclePolicyConfirmModal.noTemplateSelectedErrorMessage',
+        'xpack.indexLifecycleMgmt.policyTable.addLifecyclePolicyToTemplateConfirmModal.noTemplateSelectedErrorMessage',
```

Comment:
@maryia-lapata Please note I've changed a lot of the IDs in this file. Will this impact the translations?